### PR TITLE
UI

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
added UI
added basic routing

### CODE REVIEW

**Review by Nishant-96 — APPROVED** [review]
Liked the UI
Suggestion: maybe add some padding right on the follow button in Suggestion.

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      px="4"
+      bg={useColorModeValue("white", "gray.800")}
+      borderBottomWidth="1px"
+      borderColor={useColorModeValue("inherit", "gray.700")}
+      h="14"
+      w="full"
+    >
+      <Flex
+        // position="fixed"
+        // sx={{ zIndex: 3 }}
```
Comment (Pratik1005) [inline src/components/Navbar.jsx:34]: remove commented code if not needed

File: src/theme/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  initialColorMode: "light",
+  useSystemColorMode: false,
+};
+// const breakpoints = {
+//   sm: '550px',
+//   md: '768px',
+//   lg: '960px',
+//   xl: '1200px',
+//   '2xl': '1536px',
+// }
```
Comment (Pratik1005) [inline src/theme/index.js:40]: same here

**Review by Pratik1005 — APPROVED** [review]
Great UI, good to merge

**Review by saketkum11 — APPROVED** [review]
- clean UI
- much easy to understand
- reset sugguestion are mentioned above.

**Pull request merged**